client/src/js/references/components/Detail/Targets/Targets.js

```diff
@@ -1,28 +1,26 @@
 import React from "react";
 import styled from "styled-components";
 import { connect } from "react-redux";
-import { map, remove } from "lodash-es";
+import { map, reject, remove } from "lodash-es";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['remove' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4993528)